Dangerfile

```diff
@@ -16,25 +16,46 @@ module VSPDanger
     end
   end
 
+  class Result
+    ERROR = :error
+    WARNING = :warning
+    SUCCESS = :success
```

Comment:
Why are these extracted to constants?

Reply:
to prevent errors and troubleshooting issues with "magic strings".